src/coreclr/nativeaot/BuildIntegration/WindowsAPIs.txt

```diff
@@ -1310,8 +1310,6 @@ kernel32!QueryFullProcessImageNameW
 kernel32!QueryIdleProcessorCycleTime
 kernel32!QueryIdleProcessorCycleTimeEx
 kernel32!QueryMemoryResourceNotification
-kernel32!QueryPerformanceCounter
```

Comment:
Why are deleting this from here? It should stay.

Reply:
I thought this is for things we are directly using. Indirectly we use a lot more, I think.

Reply:
It is still fine for managed code to call QPC directly and no go through the PAL.

This change just makes calls to QPC from managed calls a bit slower.

Reply:
I misunderstood the purpose of this file. I will revert the change.